tests/bin/getuid.c

```diff
@@ -0,0 +1,8 @@
+#include "libc.h"
+
+int main(void) {
+    if(!is_enarx()) {
+        return 0;
+    }
```

Comment:
```suggestion
    if (!is_enarx())
        return 0;
```

Reply:
In general, I prefer the original version, because it's better for patches.

Reply:
I'll accept the braces. I require a space after `if`, however.

Reply:
Done.